code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -701,25 +719,25 @@ nobiliumsuppression = INFINITY
 	min_requirements = list(
 		/datum/gas/nitrogen = 10,
 		/datum/gas/tritium = 5,
-		"TEMP" = 5000000)
+		"TEMP" = 50000)
```

Comment:
Why the change?

Reply:
so, to make nob you need fusion heat, there is no other way around it. Now there is a var named energy_taken, that is responsible to cool down the mix when making this gas and it accounts the amount of bz inside. Now at fusion heat (1e7, 1e8) this amount is so small that you wouldn't even notice that the heat is going down so you can make the gas almost without thinking. By making the lower temp 50K you will be able to make a burning canister and use it to make this gas faster than making fusion first, but with the dropdown that you have to add BZ otherwise the gas will just cool down too much to be able to do something with it (all of this is thinked for open air reactions, most of reactions i changed work best in open air and worst in canisters)

Reply:
Alright, sounds good to me. Just like the rest, keep an eye on things